scalingo/main.go

```diff
@@ -22,25 +22,19 @@ var (
 )
 
 func DefaultAction(c *cli.Context) error {
```

Comment:
Can you also make this function private?
```suggestion
func defaultAction(c *cli.Context) error {
```